.github/workflows/build.yml

```diff
@@ -0,0 +1,172 @@
+name: Build
+
+on:
+  push:
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  build-web:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        name: Clone
+      - uses: actions/setup-node@v1
+        name: Setup Node
+        with:
+          node-version: "*"
```

Comment:
I would prefer building with LTS-releases, i.e., `lts/*`. Additionally, would it make sense to activate `cache: yarn` here and in the following `setup-node` calls?

Reply:
Should we build with multiple versions of node? This should not be necessary since we don't use node as a runtime, merely as a build tool (which should create the same artifact regardless of the node version).

Reply:
I think a single version (newest LTS) is sufficient in this case